code/src/Components/FavoriteFood.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import './favoriteFood.css'
+
+export const FavoriteFood = (props) => {
+  const {food, setFood} = props
+
+  return (
+    <label className="question">
+      <h3>What is your favorite food?</h3>
+      <input
+        type='text'
+        onChange={event => setFood(event.target.value)}
+        value={food}
+        className="text-input"
+        placeholder="Omnomnom"
```

Comment:
Here you are using both `"` and `'` for the same things - would be great if you choose one and stick to it